### ISSUE TITLE
Add "Why might this be useful?" meta information for each guide

### ISSUE THREAD

**Issue by urcades** [new-computers/guides#5]
On https://guides.newcomputers.group/, we should eventually include a "why this might be useful" or "why does it matter if I set this up?" or "what's the point?" section to each guide.

Clearly laying out a benefit or life-value for each of these nascent platforms is likely good for people who are less technical, and less able to immediately understand why these projects matter.

**Comment by callil** [comment]
yes - and maybe even a standardized md guide example. for instance - should each guide have a mandatory video?

**Comment by edouerd** [comment]
omfg, definitely video — idk if we should establish a stylistic standard for those, but I think we could have a lot of fun with them
